src/cljs/rems/application.cljs

```diff
@@ -594,6 +594,47 @@
              [:option {:value key}
               (get label language key)]))]))
 
+(defn- encode-multiple-values [values]
+  (->> values
+       sort
+       (str/join " ")))
+
+(defn- decode-multiple-values [value]
+  (-> value
+      (str/split #"\s+")
+      set
+      (disj "")))
```

Comment:
Maybe you could name this empty string step as the default or unselected?

Reply:
That's not really what the empty string is. It's the garbage left over by the split operation when there is leading or trailing space or the set is empty.